### PR TITLE
docs: update live demo for change detector

### LINKED ISSUE #44553: Live demo in ChangeDetectorRef documentation is outdated
### Description

The [live demo](https://plnkr.co/edit/GC512b?p=preview) for [ChangeDetectorRef#use-markforcheck-with-checkonce-strategy](https://angular.io/api/core/ChangeDetectorRef#use-markforcheck-with-checkonce-strategy) is using the deprecated Angular version `2.0.0-alpha.37`.

Related to #21525, #22370.

### What is the affected URL?

https://angular.io/api/core/ChangeDetectorRef#use-markforcheck-with-checkonce-strategy

### PR DESCRIPTION
**Pull request by naaajii**
fixes #44553

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

Update the link for [markForCheck](https://angular.io/api/core/ChangeDetectorRef#use-markforcheck-with-checkonce-strategy) strategy live demo. 

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

It redirects you to some stone age Angular version

Issue Number: #44553

## What is the new behavior?

Updated stackblitz link for markForCheck demo.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

#44553 also mentions about this [issue](https://github.com/angular/angular/blob/a92a89b0eb127a59d7e071502b5850e57618ec2d/packages/core/src/render3/STATUS.md?plain=1#L7), tho I'm not sure how to go along with it

### CODE REVIEW

**Review by jessicajaniuk — APPROVED** [review]
LGTM

**Review by jessicajaniuk — APPROVED** [review]

File: packages/core/src/change_detection/change_detector_ref.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ import {ViewRef} from '../render3/view_ref';
  *
  * The following example sets the `OnPush` change-detection strategy for a component
  * (`CheckOnce`, rather than the default `CheckAlways`), then forces a second check
- * after an interval. See [live demo](https://plnkr.co/edit/GC512b?p=preview).
+ * after an interval. See [live demo](https://stackblitz.com/edit/angular-myjuxd?file=src/main.ts).
```
Comment (AndrewKushnir) [inline packages/core/src/change_detection/change_detector_ref.ts:36]:
```suggestion
 * after an interval.
```

**Review by AndrewKushnir — COMMENTED** [review]
@Naaajii thanks for addressing the feedback. Could you please also remove the link to the live demo? We have an example below, so it should be sufficient. The Stackblitz that you've created is something that we wouldn't be able to update later (due to permissions), so we can drop the link (in favor of an example below in the guide).

File: packages/core/src/change_detection/change_detector_ref.ts (hunk 1 above)
Comment (Naaajii) [reply packages/core/src/change_detection/change_detector_ref.ts]: Sure, I've reverted the previous code for example & remove the plnkr link.

Comment (AndrewKushnir) [reply packages/core/src/change_detection/change_detector_ref.ts]: Thanks!

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit bb4875682d5b280d6b7433596c025ec555d77ef7.

**Pull request closed without merging**